session/syncedTemplate.go

```diff
@@ -125,3 +127,39 @@ func (input *SyncedTemplate) ReplaceWithoutSessionVariables(data interface{}) (s
 	}
 	return buf.String(), nil
 }
+
+func add(iVal1 interface{}, iVal2 interface{}) (int64, error) {
+	val1, err := parseToInt64(iVal1)
+	if err != nil {
+		return 0, err
+	}
+	val2, err := parseToInt64(iVal2)
+	if err != nil {
+		return 0, err
+	}
+
+	return val1 + val2, nil
+}
+
+func parseToInt64(val interface{}) (int64, error) {
+	switch val := val.(type) {
+	case string:
+		val2, err := strconv.ParseInt(val, 10, 64)
+		if err != nil {
+			return 0, errors.Wrapf(err, "string<%s> not parseable to integer", val)
+		}
+		return val2, nil
+	case int:
```

Comment:
Use a case list to avoid repeating the same line

Reply:
not possible due to the case to int64

Reply:
e.g. this is *not*  valid syntax due to the cast.
```golang
	case int, int8:
		return int64(val), nil

```

Reply:
This is due to although they look the same to us, they don't look the same to the compiler, it'll see it as
```golang
case int:
   return int64(int)
 case int8, int16:
   return int64(interface{})
```

Reply:
🤯